gluster-block.spec.in

```diff
@@ -77,12 +77,19 @@ rm -rf ${RPM_BUILD_ROOT}
 %{_unitdir}/gluster-blockd.service
 %{_unitdir}/gluster-block-target.service
 %else
-%attr(755, root, root) %{_initddir}/gluster-blockd
+%attr(0644,-,-) %{_initddir}/gluster-blockd
```

Comment:
This will be located in /etc/init.d/ as one executable command, not the systemd service. And the executable mode is needed.
Thanks,

Reply:
Done :-)